config/interfaces.go

```diff
@@ -0,0 +1,15 @@
+package config
+
+// InterfacesConfig configuration for blocky's interfaces (REST, DoH, ...)
+type InterfacesConfig struct {
+	Rest RESTConfig `yaml:"rest"`
+	DoH  DoHConfig  `yaml:"dns-over-http"`
+}
```

Comment:
Would it be sensible to add another option like `debug-api` to disable the `/debug` endpoint?
Or do you consider it out of scope and should be added later on?